### PR TITLE
[1.13.x-blue] [KOGITO-7457] - Rebranding changes on the kogito-operator repository …

### PR DESCRIPTION
**Pull request by davidesalerno**
…(cont.)

Signed-off-by: Davide Salerno <dsalerno@redhat.com>

See: [KOGITO-7457](https://issues.redhat.com/browse/KOGITO-7457)

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/main/README.md#contributing-to-the-kogito-operator)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains a link to the JIRA issue
- [x] Pull Request contains a description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've ran `make before-pr` and everything is working accordingly
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](https://github.com/kiegroup/kogito-operator/blob/main/RELEASE_NOTES.md) entry regarding this change

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Run operator BDD testing</b>
  Please add comment: <b>/jenkins test</b>

* <b>Run RHPAM operator BDD testing</b>
  Please add comment: <b>/jenkins RHPAM test</b>
</details>

### CODE REVIEW

File: config/manifests/rhpam/bases/rhpam-kogito-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ metadata:
     capabilities: Basic Install
     categories: Integration & Delivery
     certified: "true"
-    containerImage: registry.stage.redhat.io/rhpam-7/rhpam-kogito-rhel8-operator:0.0.0
+    containerImage: registry.connect.redhat.com/ibm-bamoe/bamoe-kogito-rhel8-operator:0.0.0
     description: IBM BAMOE Kogito Operator for deployment and management of Kogito
       services.
     operators.openshift.io/valid-subscription: '["Red Hat Process Automation Manager",
```
[the remark this answers is not in the document]
Comment (radtriste) [reply config/manifests/rhpam/bases/rhpam-kogito-operator.clusterserviceversion.yaml:12]: that is something we would need if we are building the bundle

**Comment by radtriste** [comment]
I think the `com.ibm.component` labels need to be set back to `com.redhat.component`
The was an issue on images ...

File: bundle/rhpam/manifests/rhpam-kogito-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -57,17 +57,15 @@ metadata:
     capabilities: Basic Install
     categories: Integration & Delivery
     certified: "true"
-    containerImage: registry.stage.redhat.io/rhpam-7/rhpam-kogito-rhel8-operator:8.0.0
+    containerImage:  registry.connect.redhat.com/ibm-bamoe/bamoe-kogito-rhel8-operator:8.0.0
```
Comment (spolti) [inline bundle/rhpam/manifests/rhpam-kogito-operator.clusterserviceversion.yaml:60]:
```suggestion
    containerImage: registry.connect.redhat.com/ibm-bamoe/bamoe-kogito-rhel8-operator:8.0.0
```

File: config/manifests/rhpam/bases/rhpam-kogito-operator.clusterserviceversion.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (spolti) [reply config/manifests/rhpam/bases/rhpam-kogito-operator.clusterserviceversion.yaml:12]: Does osbs requires it? otherwise we can remove.

**Review by spolti — CHANGES_REQUESTED** [review]

File: rhpam-operator.yaml [hunk 3]
```diff
@@ -2452,7 +2452,7 @@ spec:
         - --upstream=http://127.0.0.1:8080/
         - --logtostderr=true
         - --v=10
-        image: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.10.0
+        image: registry.connect.redhat.com/openshift4/ose-kube-rbac-proxy:v4.10.0
```
[the remark this answers is not in the document]
Comment (davidesalerno) [reply rhpam-operator.yaml]: You are right, thanks!

File: config/manifests/rhpam/bases/rhpam-kogito-operator.clusterserviceversion.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (spolti) [reply config/manifests/rhpam/bases/rhpam-kogito-operator.clusterserviceversion.yaml]: @Ginxo ^

File: bundle/rhpam/manifests/bamoe-kogito-operator.clusterserviceversion.yaml [hunk 4]
```diff
@@ -983,8 +981,8 @@ spec:
   - native
   - automation
   labels:
-    alm-owner-kogito: rhpam-kogito-operator
-    operated-by: rhpam-kogito-operator.8.0.0-1
+    alm-owner-kogito: bamoe-kogito-operator
+    operated-by: bamoe-kogito-operator.0.0.0
```
Comment (desmax74) [inline bundle/rhpam/manifests/bamoe-kogito-operator.clusterserviceversion.yaml:985]: 0.0.0 ?

Comment (spolti) [reply bundle/rhpam/manifests/bamoe-kogito-operator.clusterserviceversion.yaml:985]: it is a place holder, it gets updated when you run make bundle.

File: Makefile.rhpam [hunk 5]
```diff
@@ -6,7 +6,7 @@ DEFAULT_CHANNEL = 8.x
 BUNDLE_CHANNELS = --channels=$(CHANNELS)
 BUNDLE_DEFAULT_CHANNEL = --default-channel=$(DEFAULT_CHANNEL)
 
-IMAGE_TAG_BASE = registry.stage.redhat.io/rhpam-7/rhpam-kogito-rhel8-operator
+IMAGE_TAG_BASE =  registry.connect.redhat.com/ibm-bamoe/bamoe-kogito-rhel8-operator
```
[the remark this answers is not in the document]
Comment (davidesalerno) [reply Makefile.rhpam]: Thanks Max, space removed

File: examples/rhpam/springboot.Dockerfile [hunk 6]
```diff
@@ -1,4 +1,4 @@
-FROM registry.redhat.io/rhpam-7/rhpam-kogito-runtime-jvm-rhel8:7.12.0
+FROM ibm-bamoe/bamoe-kogito-runtime-jvm-rhel8:7.12.0
```
Comment (sutaakar) [inline examples/rhpam/springboot.Dockerfile:1]: Shouldn't it be `registry.connect.redhat.com/ibm-bamoe/bamoe-kogito-runtime-jvm-rhel8:8.0.0`?

Comment (davidesalerno) [reply examples/rhpam/springboot.Dockerfile:1]: +1

**Comment by radtriste** [comment]
sorry there was an issue with my commenting. I can see those have been solved

File: bundle/rhpam/metadata/annotations.yaml [hunk 7]
```diff
@@ -4,8 +4,8 @@ annotations:
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: rhpam-kogito-operator
```
Comment (radtriste) [inline bundle/rhpam/metadata/annotations.yaml:6]: should we update that one ?

**Review by radtriste — COMMENTED** [review]
one final comment from my side

File: bundle/rhpam/metadata/annotations.yaml (hunk 7 above)
Comment (davidesalerno) [reply bundle/rhpam/metadata/annotations.yaml:6]: It makes sense and it seems safe: I'm doing it

**Review by spolti — APPROVED** [review]

File: bundle/rhpam/metadata/annotations.yaml [hunk 8]
```diff
@@ -4,9 +4,9 @@ annotations:
...
```
Comment (radtriste) [inline bundle/rhpam/metadata/annotations.yaml:6]: did you make the change ?

Comment (davidesalerno) [reply bundle/rhpam/metadata/annotations.yaml:6]: @radtriste yes, but when I run the make bundle command it seems to be overridden but I cannot underderstand where is managed that annotation, I suspect in the Makefile package -> https://github.com/kiegroup/kogito-operator/blob/main/Makefile.rhpam#L66

Comment (spolti) [reply bundle/rhpam/metadata/annotations.yaml:6]: @radtriste @davidesalerno I'll be merging it, if needed we can revisit later. probably new changes will be needed in the coming days.

**Pull request merged**